src/main/resources/default_browser_config.json

```diff
@@ -51,11 +55,19 @@
     "verboseLogging": true,
     "headlessOptions": {
       "headless": true
-    }
+    },
+    "tab1" : "iPad",
```

Comment:
though we have added the values for firefox and safari, this is not yet implemented in Drivers, right?

Reply:
yes, the logic is only present for Chrome as of now